molgenis-platform-integration-tests/src/test/java/org/molgenis/integrationtest/data/transaction/TransactionTestConfig.java

```diff
@@ -0,0 +1,29 @@
+package org.molgenis.integrationtest.data.transaction;
+
+import org.molgenis.data.populate.IdGenerator;
+import org.molgenis.data.populate.IdGeneratorImpl;
+import org.molgenis.data.transaction.PostgreSqlTransactionManager;
+import org.molgenis.data.transaction.TransactionExceptionTranslatorRegistry;
+import org.molgenis.data.transaction.TransactionManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+import javax.sql.DataSource;
+
+@Configuration
+@Import({ TransactionExceptionTranslatorRegistry.class })
+public class TransactionTestConfig
+{
+	@Autowired
+	private DataSource dataSource;
+
+	@Bean
+	public TransactionManager transactionManager()
+	{
+		IdGenerator idGenerator = new IdGeneratorImpl();
```

Comment:
Add IdGeneratorImpl to Import list, Autowire in TransactiontestConfig and use in PostgreSqlTransactionManager constructor. This has the advantage that your test uses one IdGenerator instead of two (possible different) generators.